### PR TITLE
Added template json for Travel template

### PR DESCRIPTION
**Pull request by mariano-formidable**
This PR adds a json file that describes a static (no animations) version of the Travel template.  Using our `PreviewPage` component I was able to get the template to render inside storybook.

The static Travel template is about 95% complete, a few small details are still missing but I figure this is a good start to test out creating a template using our editor data structure, and it unblocks us so we can start creating the Template Dashboard pages.  It also gives us a data structure to add in animations.  

image
(Travel template in storybook, scroll down in storybook to see the rest if the pages)

### CODE REVIEW

File: assets/src/dashboard/app/api/apiProvider.js [hunk 1]
```diff
@@ -57,6 +57,10 @@ export default function ApiProvider({ children }) {
 
   const fetchStories = useCallback(
     async ({ status = STORY_STATUSES[0].value }) => {
+      if (!api.stories) {
+        return [];
+      }
```
Comment (mariano-formidable) [inline assets/src/dashboard/app/api/apiProvider.js:59]: Storybook uses the `ApiProvider` but doesn't pass in a config, so I've added these checks so we don't make fetch requests on undefined routes.

File: assets/src/dashboard/templates/stories/travel.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  ]).isRequired,
+};
+
+const TemplateFonts = () => (
+  <link
+    rel="stylesheet"
+    id="google-sans-font-css"
+    href="https://fonts.googleapis.com/css2?family=Lora&family=Oswald:wght@500&family=Playfair+Display&display=swap"
+    media="all"
+  />
```
Comment (mariano-formidable) [inline assets/src/dashboard/templates/stories/travel.js:71]: Couldn't find a clean way to properly load up the fonts in storybook, so I just linked in the specific fonts used for the Travel template!  Works like a charm!

**Review by carloskelly13 — APPROVED** [review]

**Review by BrittanyIRL — APPROVED** [review]
Looks good! 
Eventually it'd be great to move the `StoryProvider` to be reused by other storybook cases where we need it, but I don't think that's necessary for this.

**Comment by mariano-formidable** [comment]
> Looks good!
> Eventually it'd be great to move the `StoryProvider` to be reused by other storybook cases where we need it, but I don't think that's necessary for this.

Yes, I agree!  My thinking was as soon as I make a second template then I'll move `StoryProvider` out to a shared location so both templates can use it.

**Pull request merged**